apps/shell/tash_main.c

```diff
@@ -55,6 +55,10 @@ enum tash_input_state_e {
 #define TASH_PROMPT           "TASH>>"
 #endif							/* CONFIG_TASH */
 
+#ifdef CONFIG_PM
+#define TASH_PM_TIMEDSUSPEND_TIME_IN_MS 1000
```

Comment:
After input the enter key, can we input next always in 1s? Sometimes yes, but sometimes no. Let me suggest 5s. It's the TASH so that 1s -> 5s does not affect power consumption almost. And because it's user intend so that it's ok with more power consumption.

Reply:
Ok I will change it 5s